topics/data-documentation.qmd

```diff
@@ -0,0 +1,74 @@
+# Data Documentation
+By creating documentation about your research data you can make it easier for yourself or for others to manage, find, assess and use your data. The process of documenting means to describe your data and the methods by which they were collected, processed and analysed. The documentation or descriptions are also referred to as metadata, i.e. data about data. These metadata can take various forms and can describe data on different levels.
+
+An example that is frequently used to illustrate the importance of metadata is the use of the label on a can of soup. The label tells you what kind of soup the can contains, what ingredients are used, who made it, when it expires and how you should prepare the soup for consumption.
+
+When you are documenting data, you should take into account that there are different kinds of metadata and that these metadata are governed by various standards. These include, but are not limited to:
+
+1.	FAIR data principles: a set of principles to make data Findable, Accessible, Interoperable and Reusable. 
+2.	Guidelines for unstructured metadata: mostly research domain-specific guidelines on how to create READMEs or Codebooks to describe data.
+3.	Standards for structured metadata: generic or research domain-specific standards to describe data.
+
+![](../public/data_documentation.png)
+
+## FAIR data principles
+The FAIR data principles provide guidelines to improve the Findability, Accessibility, Interoperability, and Reuse of digital assets. The principles emphasise machine-actionability, i.e., the capacity of computational systems to find, access, interoperate, and reuse data with none or minimal human intervention. 
+
+More information can be found in the section about the [FAIR data principles](https://ubvu.github.io/open-handbook/topics/fair-principles.html).
+
+## Unstructured metadata
+Unstructured metadata are mainly intended to provide more detailed information about the data and is primarily readable for humans. The type of research and the nature of the data influence what kind of unstructured metadata is necessary. Unstructured metadata are attached to the data in a file. The format of the file is chosen by the researcher. 
+
+### README
+A README file provides information about data and is intended to ensure that data can be correctly interpreted, by yourself or by others. A README file must be submitted along with the data.
+
+Example of READMEs
+- [Guidelines for creating a README file – 4TU.ResearchData](https://data.4tu.nl/s/documents/Guidelines_for_creating_a_README_file.pdf)
+- [Guide to writing "readme"-style metatada - Cornell Data Services](https://data.research.cornell.edu/data-management/sharing/readme/)
```

Comment:
```suggestion
Example of READMEs

- [Guidelines for creating a README file – 4TU.ResearchData](https://data.4tu.nl/s/documents/Guidelines_for_creating_a_README_file.pdf)
- [Guide to writing "readme"-style metatada - Cornell Data Services](https://data.research.cornell.edu/data-management/sharing/readme/)
```

As far as I know the list is fine with either `-` or `*` in Markdown (although I did not check with respect to `.qmd` files). Important is to have a whiteline before and after.

Reply:
Thank you, Chris! I've used `-` in Markdown before, indeed, and I was surprised it didn't work.